### DISCUSSION TITLE
Admob firebase error "Invoke Firebase method getInstance error"

### DISCUSSION THREAD

**Discussion by angelystor** [Poing-Studios/Godot-AdMob-Android-iOS#182] [Q&A]
I'm trying to set up admob and am having this issue, seems like Ads requires a newer Firebase SDK but it's not installed. I'm running it on a virtual device at the moment to test, but this error doesn't seem related to that.
Any help?

![image](https://github.com/Poing-Studios/godot-admob-android/assets/524826/55c81442-ba36-4150-8f41-7cc605a465d8)

**Comment by gumaciel** [comment]
Hi @angelystor, we are not using anything about "Firebase" on our plugin.

I've research here and this message may be normal to appear and will not affect your ads: https://groups.google.com/g/google-admob-ads-sdk/c/kAAhkzmxbIk/m/op5RGLCZDAAJ

`The error that you have noticed is only specific to Firebase and will not impact the ads from being served in your app. You will be able to fetch live ads to your app without any issues.`
